cli/command/manifest/cmd.go

```diff
@@ -0,0 +1,45 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+
+	"github.com/spf13/cobra"
+)
+
+// NewManifestCommand returns a cobra command for `manifest` subcommands
+// nolint: interfacer
+func NewManifestCommand(dockerCli *command.DockerCli) *cobra.Command {
+	// use dockerCli as command.Cli
+	cmd := &cobra.Command{
+		Use:   "manifest COMMAND",
+		Short: "Manage Docker image manifests and lists",
+		Long:  manifestDescription,
+		Args:  cli.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Fprintf(dockerCli.Err(), "\n"+cmd.UsageString())
```

Comment:
Just happened to notice: This should be `fmt.Fprintf(dockerCli.Err(), "\n%s", cmd.UsageString())` or `fmt.Fprint(dockerCli.Err(), "\n"+cmd.UsageString())`. Otherwise a % character in the usage message would cause format string problems.